include missing attributes in mails

NobodysNightmare opened the pull request:
## OpenProject work package

https://community.openproject.org/work_packages/19771
## Description

This PR adds missing attributes (e.g. dates) to notification e-mails.

NobodysNightmare committed:
include missing attributes in mails

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://nobodysnightmare-feature-show-dates-in-mail.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://nobodysnightmare-feature-show-dates-in-mail.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

ulferts commented on app/views/user_mailer/_issue_details.text.erb:
```diff
@@ -37,6 +37,13 @@ See doc/COPYRIGHT.rdoc for more details.
+
+<% if Setting.work_package_done_ratio != 'disabled' %>
+  <%= WorkPackage.human_attribute_name(:done_ratio)    %>: <%= issue.done_ratio    %>
+<% end %>
```
@NobodysNightmare this will create a blank line between estimated_hours and done_ratio which shouldn't be necessary. BTW: This is probably also true for the custom fields.

NobodysNightmare committed:
fix whitespacing in '_issue_details.text.erb'

empty lines and indentation used for code formatting is immediately visible in the resulting output.

I tried to employ some special rules to at least allow for usage of indentation.
